Kajabi/sage-lib
# Carousel Pattern

## Description

Create an easily copyable pattern for developers to reference for the Next Best Actions carousel feature.


## Screenshots
![Screen Shot 2021-10-29 at 4 24 27 PM](https://user-images.githubusercontent.com/24237393/139503413-2f21c584-e4f2-4572-899f-1656529107d0.png)


## Testing in `sage-lib`
1. Navigate to http://localhost:4000/pages/patterns/next_best_actions
2. Verify that code is rendering
3. Verify that documentation appears complete


## Testing in `kajabi-products`
1. (**LOW**) Creates a pattern for Next Best Actions in Sage. No `kajabi-products` testing necessary.
 
## Related
Closes #866
